GI05MOAS-PG566/GI05MOAS-PG566_R00003_ingest.csv

```diff
@@ -0,0 +1,8 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+#mi.dataset.driver.moas.gl.engineering.glider_eng_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-PG566/R00003/merged/gi_*.mrg,GI05MOAS-PG566-00-ENG000000,recovered_host,Not Expected,short deployent; no data expected
+#mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-PG566/R00003/merged/gi_*.mrg,GI05MOAS-PG566-01-CTDGVM000,recovered_host,Not Expected,short deployent; no data expected
+#mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-PG566/R00003/merged/gi_*.mrg,GI05MOAS-PG566-02-DOSTAM000,recovered_host,Not Expected,short deployent; no data expected
+#mi.dataset.driver.moas.gl.flort_m.flort_m_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-PG566/R00003/merged/gi_*.mrg,GI05MOAS-PG566-03-FLORTM000,recovered_host,Not Expected,short deployent; no data expected
+#mi.dataset.driver.moas.gl.flort_o.flort_o_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-PG566/R00003/merged/gi_*.mrg,GI05MOAS-PG566-04-FLORTO000,recovered_host,Not Expected,short deployent; no data expected
+#mi.dataset.driver.nutnr_m.glider.nutnr_m_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-PG566/R00003/merged/gi_*.mrg,GI05MOAS-PG566-05-NUTNRM000,recovered_host,Not Expected,short deployent; no data expected
+#mi.dataset.driver.moas.gl.parad.parad_m_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-PG566/R00003/merged/gi_*.mrg,GI05MOAS-PG566-06-PARADM000,recovered_host,Not Expected,short deployent; no data expected
```

Comment:
Looks good.  Glider recovered right after deployment and re-deployed days later.